src/CoreWCF.WebHttp/src/CoreWCF/Dispatcher/FormatSelectingMessageInspector.cs

```diff
@@ -0,0 +1,224 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Mime;
+using CoreWCF.Channels;
+using CoreWCF.Description;
+using CoreWCF.Runtime;
+using CoreWCF.Web;
+
+namespace CoreWCF.Dispatcher
+{
+    internal class FormatSelectingMessageInspector : IDispatchMessageInspector
+    {
+        static readonly IEnumerable<string> s_wildcardMediaTypes = new List<string>() { "application", "text" };
+        readonly List<MultiplexingFormatMapping> _mappings;
```

Comment:
Accessibility modifiers for multiple members

Reply:
This is addressed in e575590ad880fad2afb332b5bf532abdcfbbd23b.